src/components/AgreementSections/CoveredEResourcesList.js

```diff
@@ -58,9 +59,20 @@ export default class CoveredEResourcesList extends React.Component {
     platform: e => get(e._object, 'pti.platform.name', '-'),
     package: e => get(e._object, 'pkg.name', '-'),
     coverage: e => <CoverageStatements statements={e.coverage} />,
-    isCustomCoverage: e => (e.customCoverage ? <CustomCoverageIcon /> : ''),
     accessStart: e => this.renderDate(get(e._object, 'accessStart')),
     accessEnd: e => this.renderDate(get(e._object, 'accessEnd')),
+    isCustomCoverage: line => {
+      if (!line.customCoverage) return '';
+      return (
+        <Tooltip
+          id="custom-coverage-tooltip"
```

Comment:
Just realised that this `id` won't be unique across the page while it should. So maybe something like this:
```suggestion
          id={`covered-eresources-cc-tooltip-${line.rowIndex}`}
```

(`rowIndex` is provided by `MultiColumnList`)